libs/components/text-editor/src/lib/modules/text-editor/toolbar/text-editor-toolbar.component.ts

```diff
@@ -36,87 +37,91 @@ import { UrlTarget } from '../url-modal/text-editor-url-target';
 })
 export class SkyTextEditorToolbarComponent implements OnInit {
   @Input()
-  public editorFocusStream = new Subject();
+  public set editorFocusStream(value: Subject<void>) {
+    this.#_editorFocusStream = value;
+    this.#subscribeEditorFocus();
+  }
 
-  @Input()
-  public editorId: string;
+  public get editorFocusStream(): Subject<void> {
+    return this.#_editorFocusStream;
+  }
 
   @Input()
-  public fontList: SkyTextEditorFont[];
+  public fontList: SkyTextEditorFont[] = [];
 
   @Input()
-  public fontSizeList: number[];
+  public fontSizeList: number[] = [];
 
   @Input()
-  public toolbarActions: SkyTextEditorToolbarActionType[];
+  public toolbarActions: SkyTextEditorToolbarActionType[] = [];
```

Comment:
☝️ Do these inputs need `| undefined`?

Reply:
I intentionally left out `| undefined` on the toolbar and menubar components because they're internal. Since we control when the inputs are set, we can guarantee they're never set to `undefined`.